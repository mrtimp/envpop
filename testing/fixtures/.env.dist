# comment

COMMENT="VALUE" # comment
DYNAMIC_COMMENT_STRING= # comment

STRING=VALUE
EMPTY_STRING=
STRING_QUOTE='VALUE'
STRING_DOUBLE_QUOTE="VALUE"
DYNAMIC_STRING=

NUMBER=123
NUMBER_AS_STRING_QUOTE='123'
NUMBER_AS_STRING_DOUBLE_QUOTE="123"
DYNAMIC_NUMBER=

FLOAT=1.23
FLOAT_AS_STRING_QUOTE='1.23'
FLOAT_AS_STRING_DOUBLE_QUOTE="1.23"
DYNAMIC_FLOAT=

BOOLEAN_TRUE=true
BOOLEAN_FALSE=false
DYNAMIC_BOOLEAN=

NULL=null
NULL_AS_STRING_QUOTE='null'
NULL_AS_STRING_DOUBLE_QUOTE="null"
DYNAMIC_NULL=
