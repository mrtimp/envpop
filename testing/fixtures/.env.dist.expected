# comment

COMMENT="VALUE" # comment
DYNAMIC_COMMENT_STRING="VALUE" # comment

STRING=VALUE
EMPTY_STRING=
STRING_QUOTE='VALUE'
STRING_DOUBLE_QUOTE="VALUE"
DYNAMIC_STRING="VALUE"

NUMBER=123
NUMBER_AS_STRING_QUOTE='123'
NUMBER_AS_STRING_DOUBLE_QUOTE="123"
DYNAMIC_NUMBER=123

FLOAT=1.23
FLOAT_AS_STRING_QUOTE='1.23'
FLOAT_AS_STRING_DOUBLE_QUOTE="1.23"
DYNAMIC_FLOAT=1.23

BOOLEAN_TRUE=true
BOOLEAN_FALSE=false
DYNAMIC_BOOLEAN=true

NULL=null
NULL_AS_STRING_QUOTE='null'
NULL_AS_STRING_DOUBLE_QUOTE="null"
DYNAMIC_NULL=null
